packages/core/src/conversation/ConversationService/Utility/isConversationEvent.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {BackendEvent, ConversationEvent} from '@wireapp/api-client/lib/event';
+
+const CONVERSATION_EVENT_CATEGORY = 'conversation';
+
+export const isConversationEvent = (event: BackendEvent): event is ConversationEvent => {
+  const [category] = event.type.split('.');
+  return category === CONVERSATION_EVENT_CATEGORY;
+};
```

Comment:
I know we do those kind of checks in the webapp, but I think they actually can be harmful. 
We don't have any guarantee that there are events in the future that do not start with `conversation` that still need to be processed by the conversation service. 

Let's not try to filter events before hands, let's just react at all the ones we want to without any pre-filter. This is going to scale way better in the future :)